empress/recon_vis/recon_viewer.py

```diff
@@ -378,7 +378,7 @@ def _sigmoid(x: float):
     :param x: A number to be plugged into the function
     :return a sigmoid value based on the input value, x
     """
-    return (1 / (1 + math.e**(-x)))
+    return (1.5 / (1 + math.e**(-x)))
```

Comment:
It might be cleaner to write `1.5 * _sigmoid(x)` on line 372 and keep line 381 the same.